src/kirby/components/list/list.component.tns.html

```diff
@@ -7,7 +7,7 @@
     (itemTap)="onItemSelect($event)">
     <!-- ITEM -->
     <ng-template tkListItemTemplate let-item="item" let-index="index">
-      <StackLayout (loaded)="onRowLoaded($event)">
+      <StackLayout kirbyElevation="z2" iosMasksToBounds="true">
```

Comment:
Couldn't this be built into the elevation directive? Or does it have side effects on other components?

Reply:
Unfortunately it has side effects, particularly on components with border-radius such as the FAB